src/test/ReactTestUtils.js

```diff
@@ -269,9 +269,8 @@ var ReactTestUtils = {
       componentType
     );
     if (all.length !== 1) {
-      throw new Error(
-        'Did not find exactly one match for componentType:' + componentType +
-        ' (found ' + all.length + ')'
+      throw new Error('Did not find exactly one match ' +
```

Comment:
Ok, then I'll add the newline after the paren for the error messages for `findRenderedDOMComponentWithClass` and `findRenderedDOMComponentWithTag` too?